### PR TITLE
Add vasculature template

### PR DESCRIPTION
**Pull request by anitacaron**
Related to https://github.com/hubmapconsortium/ccf-validation-tools/issues/120

### CODE REVIEW

**Comment by dosumis** [comment]
Converting to draft for now as we need to discuss whether adding connected_to relationships is the best approach for these.

**Review by dosumis — CHANGES_REQUESTED** [review]
* I don't think vasculature.owl should be committed
* Maybe robot templates need to be enabled in ODK to build components/vasculature_class.owl?
* Value for >AI should be full IRI (or CURIE might also work) - this looks like just string

**Comment by anitacaron** [comment]
I followed this doc to include the template. https://github.com/obophenotype/uberon/tree/master/src/templates 
Enabling `use_custom_import_module` will create `external_import.tsv` with a list of terms to import when the imports are refreshed. (found here the doc https://github.com/OBOFoundry/COB/blob/53368763cd61e871543bfb8836aee8917cab8bc2/docs/odk-workflows/UpdateImports.md#using-the-custom-import-template) Is this what you hope to get?

**Comment by dosumis** [comment]
I think this is happening because the import statement is looking for a file in components that you've committed in templates
image

I guess the make command doesn't move it into place? @matentzn  - is there a gap in the docs?

> Enabling use_custom_import_module will create external_import.tsv with a list of terms to import when the imports are refreshed. (found here the doc https://github.com/OBOFoundry/COB/blob/53368763cd61e871543bfb8836aee8917cab8bc2/docs/odk-workflows/UpdateImports.md#using-the-custom-import-template) Is this what you hope to get?

This is not relevant (we're not doing anything with imports).   I was just speculating that it might be possible to configure ODK to rebuild from robot templates & save the file to components automatically.

**Comment by matentzn** [comment]
Didnt read all your comments, just fixed the catalog (see last commit). Was that all you need from me?

**Comment by matentzn** [comment]
Remember @anitacaron catalog.xml is relative path from `src/ontology`, not `src/`.

**Comment by anitacaron** [comment]
Thanks, @matentzn! 

Do we need to add `vasculature_class.owl`, or it will be generated when running `prepare_release`?

**Comment by matentzn** [comment]
You need it, if you want it to be seen when opening uberon-edit.obo

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -53,6 +54,7 @@ treat-xrefs-as-equivalent: OG
 treat-xrefs-as-equivalent: VSAO
 treat-xrefs-as-is_a: VHOG
 import: http://purl.obolibrary.org/obo/uberon/components/mappings.owl
+import: http://purl.obolibrary.org/obo/uberon/components/vasculature_class.owl
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:57]: @anitacaron if you add this, the owl file has to exist as well! So yes, leave it checked in here as is!

File: src/templates/vasculature_class.owl [hunk 2]
```diff
@@ -81,7 +81,7 @@
                 <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/UBERON_0001142"/>
             </owl:Restriction>
         </owl:annotatedTarget>
-        <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/uberon/core#human_reference_atlas"/>
+        <oboInOwl:inSubset rdf:resource="&lt;http://purl.obolibrary.org/obo/uberon/core#human_reference_atlas&gt;"/>
```
Comment (anitacaron) [inline src/templates/vasculature_class.owl:84]: @dosumis it was already good before changing to the complete URI; somewhere it's already defined (?)

Comment (anitacaron) [reply src/templates/vasculature_class.owl:84]: I'll revert the commit.

**Review by dosumis — APPROVED** [review]

**Pull request merged**